lib/Support/BigIntSupport.cpp

```diff
@@ -1754,6 +1750,25 @@ static OperationStatus compute(
       ((quoc.digits != nullptr) != (rem.digits != nullptr)) &&
       "untested -- calling with both or neither quoc and rem");
 
+  // Signal division by zero.
+  if (compare(rhs, 0) == 0) {
+    return OperationStatus::DIVISION_BY_ZERO;
+  }
+
+  if (lhs.numDigits < rhs.numDigits) {
```

Comment:
The assumption is incorrect on `-(2n**63n)/(2n**63n)`
I need to fix that

Reply:
The easiest way would be just do `lhs.numDigits + 1 < rhs.numDigits` in both places
And then consider improving the check in a separate PR

Reply:
Are you going to publish an updated one?

Reply:
Pushed a fix
I recommend to go with #1852 first though